tests/ui/EditTask.test.Edit_Modal_HTML_snapshot_tests_should_match_snapshot_-_without_access_keys.approved.html

```diff
@@ -1,111 +1,85 @@
     <section class="tasks-modal-priority-section">
       <label for="priority-none">Priority</label>
       <div class="task-modal-priority-option-container">
         <input type="radio" id="priority-lowest" value="lowest" />
         <label for="priority-lowest">
-          <span>L</span>
-          <span class="accesskey">o</span>
-          <span>west</span>
+          <span>Lowest</span>
           <span>⏬</span>
         </label>
       </div>
       <div class="task-modal-priority-option-container">
         <input type="radio" id="priority-low" value="low" />
         <label for="priority-low">
-          <span></span>
-          <span class="accesskey">L</span>
-          <span>ow</span>
+          <span>Low</span>
           <span>🔽</span>
         </label>
       </div>
       <div class="task-modal-priority-option-container">
         <input type="radio" id="priority-none" value="none" />
-        <label for="priority-none">
-          <span></span>
-          <span class="accesskey">N</span>
-          <span>ormal</span>
-        </label>
+        <label for="priority-none"><span>Normal</span></label>
       </div>
       <div class="task-modal-priority-option-container">
         <input type="radio" id="priority-medium" value="medium" />
         <label for="priority-medium">
-          <span></span>
-          <span class="accesskey">M</span>
-          <span>edium</span>
+          <span>Medium</span>
           <span>🔼</span>
         </label>
       </div>
       <div class="task-modal-priority-option-container">
         <input type="radio" id="priority-high" value="high" />
         <label for="priority-high">
-          <span></span>
-          <span class="accesskey">H</span>
-          <span>igh</span>
+          <span>High</span>
           <span>⏫</span>
         </label>
       </div>
       <div class="task-modal-priority-option-container">
         <input type="radio" id="priority-highest" value="highest" />
         <label for="priority-highest">
-          <span>H</span>
-          <span class="accesskey">i</span>
-          <span>ghest</span>
+          <span>Highest</span>
           <span>🔺</span>
         </label>
       </div>
     </section>
     <hr />
     <section class="tasks-modal-dates-section">
-      <label for="recurrence" class="accesskey-first">Recurs</label>
+      <label for="recurrence">Recurs</label>
       <input id="recurrence" type="text" class="tasks-modal-date-input" placeholder="Try 'every day when done'" />
       <code class="tasks-modal-parsed-date">
         🔁
         <i>not recurring</i>
       </code>
-      <label for="due" class="accesskey-first">Due</label>
+      <label for="due">Due</label>
       <input id="due" type="text" class="tasks-modal-date-input" placeholder="Try 'Mon' or 'tm' then space" />
       <code class="tasks-modal-parsed-date">
         📅
         <i>no due date</i>
       </code>
-      <label for="scheduled" class="accesskey-first">Scheduled</label>
```

Comment:
Is `accesskey-first` used any more? Could it be removed from some .scss file?

Reply:
Indeed, that is why I removed it from EditTask.scss =)

Reply:
> Indeed, that is why I removed it from EditTask.scss =)

Oops - I don't know how I missed that. Thanks.